README.md

```diff
@@ -1,14 +1,18 @@
-# Safe{Core} Protocol Specs
+# Safe{Core} Protocol Specification
+
+See also:
+- [Protocol Implementation](https://github.com/5afe/safe-core-protocol)
```

Comment:
This will be move to safe-global, should be updated later

Reply:
Will keep it in mind